### PR TITLE
Fix dangling reference

### PR DESCRIPTION
**Pull request by havogt**
## Technical Description

Another dangling reference was introduced. Caught by clang-tidy.

### CODE REVIEW

File: dawn/src/dawn/IIR/Extents.h [hunk 1]
```diff
@@ -209,16 +209,16 @@ class HorizontalExtent {
   HorizontalExtent() = default;
 
   HorizontalExtent(ast::HorizontalOffset const& hOffset) {
```
Comment (havogt) [inline dawn/src/dawn/IIR/Extents.h]: No change in this file. Was it properly formatted before?

File: dawn/src/dawn/Support/ArrayRef.h [hunk 2]
```diff
@@ -77,13 +78,17 @@ class ArrayRef {
   constexpr ArrayRef(const T (&Arr)[N]) : data_(Arr), length_(N) {}
 
 /// @brief Construct an ArrayRef from a std::initializer_list
+#if DAWN_GNUC_PREREQ(9, 0, 0)
```
Comment (havogt) [inline dawn/src/dawn/Support/ArrayRef.h]: fixes a warning in older compilers about -Winit-list-lifetime doesn't exist

File: dawn/src/dawn/IIR/Extents.h (hunk 1 above)
Comment (jdahm) [reply dawn/src/dawn/IIR/Extents.h]: That may have been my enum PR. I don't think VSCode was configured correctly to auto-apply `clang-format` on save.

**Comment by havogt** [comment]
launch jenkins

**Pull request merged**